src/lib/mina/account-update.ts

```diff
@@ -875,28 +875,11 @@ class AccountUpdate implements Types.AccountUpdate {
   }
 
   hash(): Field {
-    // these two ways of hashing are (and have to be) consistent / produce the same hash
```

Comment:
finally had to remove this, because it would have meant account update hashing depends on the bindings